package.json

```diff
@@ -797,7 +804,7 @@
       {
         "command": "vscode-docker.compose.down",
         "title": "Compose Down",
-        "description": "Stops a composition of containers",
+        "description": "Stops and removes a composition of containers",
```

Comment:
Stops containers and removes containers and networks created by up.

https://docs.docker.com/compose/reference/down/

Reply:
There was a reason I modified the text from the docs to "Stops containers and removes containers and non-external networks created by Compose Up." If you read the docs, you'll see that using the flags we use does not remove all of those.  Don't want to worry users.